### PR TITLE
Add python 3.5 to tox config

### PR DESCRIPTION
**Pull request by jklukas**

### CODE REVIEW

**Comment by graingert** [comment]
nice, can you update travis as well?

**Comment by jklukas** [comment]
Updated the travis config.

File: .travis.yml [hunk 1]
```diff
@@ -1,11 +1,12 @@
 language: python
-python: 3.4
+python: 3.5
```
Comment (jklukas) [inline .travis.yml]: Keeping the python version to the latest seems like a reasonable default here.

File: CHANGES.rst [hunk 2]
```diff
@@ -10,6 +10,8 @@
 - Fix a bug where reflecting a table could raise a ``NoSuchTableError``
   in cases where its schema is not on the current ``search_path``
   (`Issue #64 <https://github.com/sqlalchemy-redshift/sqlalchemy-redshift/pull/64>`_)
+- Add python 3.5 to the list of versions for integration tests.
+  (`Issue #61 <https://github.com/sqlalchemy-redshift/sqlalchemy-redshift/pull/61>`_)
```
Comment (graingert) [inline CHANGES.rst]: probably not worth the CHANGES.rst unless you update the trove classifiers:
```python
setup(
    ...
    classifiers=[
        ...
        'Programming Language :: Python',
        'Programming Language :: Python :: 2.6',
        'Programming Language :: Python :: 2.7',
        'Programming Language :: Python :: 3',
        'Programming Language :: Python :: 3.3',
        'Programming Language :: Python :: 3.4'
        'Programming Language :: Python :: 3.5',
        ...
    ],
...
)
```

File: .travis.yml (hunk 1 above)
Comment (graingert) [inline .travis.yml]: I agree

File: CHANGES.rst (hunk 2 above)
Comment (jklukas) [inline CHANGES.rst]: It's never been clear to me whether explicitly listing supported versions is actually useful to anybody.

I guess we might as well?

Comment (graingert) [inline CHANGES.rst]: all the cool kids do it :+1:

**Comment by jklukas** [comment]
Added trove classifiers.

**Comment by graingert** [comment]
nice one, feel free to merge when this goes green

**Pull request merged**